packages/wrangler/src/api/pages/publish.tsx

```diff
@@ -245,14 +261,30 @@ export async function publish({
 				watch: false,
 				onEnd: () => {},
 				betaD1Shims: d1Databases,
+				experimentalWorkerBundle,
 			});
 			workerFileContents = readFileSync(outfile, "utf8");
```

Comment:
We talked about this offline, but just for posterity: I think it would be _neater_ if this `buildRawWorker` and `buildFunctions` functions, which take an `outfile` prop, used that prop to write the bundle to, when `experimentalWorkerBundle` is true. I know we don't because these functions are also used by `wrangler pages dev`, but it feels like it should be `buildRawWorker`/`buildFunctions`'s responsibility to write to this file when they have all this information upfront. No big stress though. I'm happy to leave as-is for now. After we ship it, we can do a bit of cleanup when we get a second.